docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-Sphinx>=2.1
-sphinx-hoverxref
-sphinx-notfound-page
-sphinx_rtd_theme
+Sphinx==3.0.1
+sphinx-hoverxref==0.2b1
+sphinx-notfound-page==0.4
+sphinx_rtd_theme==0.4.3
```

Comment:
I would not freeze versions, and instead set `>=` restrictions as before where needed. That’s how things break for every pull request when a new version breaks things, but it is also how we find out early. Since building the documentation is not something people are expected to do themselves with released versions, I think breaking early is a better tradeoff. I don’t have a strong opinion about it, though.

Reply:
I thought pinning exact versions might be controversial :smile:
I also don't have a strong preferencem but breaking early makes sense, so let's change it.